Journalator/Source/Monitor/InvestONator.lua

```
@@ -0,0 +1,81 @@
+  end
+  
+  -- Check if this item is in any investment portfolio
+  local portfolios = Journalator.InvestONator.GetAllPortfolios()
+  
+  -- Early exit if no portfolios exist
+  if not portfolios or next(portfolios) == nil then
+    return
+  end
+  
+  local totalAmount = price * quantity
+  local itemName = GetItemInfo(itemID) or "Unknown Item"
```

Luacheck (1.2.0)

[warning] accessing undefined variable 'GetItemInfo'

(W113)